spec.html

```diff
@@ -22544,16 +22544,15 @@
 
       <!-- es6num="19.1.2.2" -->
       <emu-clause id="sec-object.create">
-        <h1>Object.create ( _O_ [ , _Properties_ ] )</h1>
+        <h1>Object.create ( _O_, _Properties_ )</h1>
```

Comment:
I guess if we consider `Object.create` a legacy function along with those others, then it'd be fine.